arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1862,25 +1863,35 @@ Result ClusterInfo::createCollectionsCoordinator(std::string const& databaseName
                                               AgencyPrecondition::Type::EMPTY, true));
       }
     }
+    
+    // additionally ensure that no such collectionID exists yet in Plan/Collections
+    precs.emplace_back(AgencyPrecondition("Plan/Collections/" + databaseName + "/" + info.collectionID,
```

Comment:
I think this cannot happen with the way we generate the IDs right now.
However it is a good precondition here, to be safe in the future

Reply:
That is correct but you never know...